bin/build-constraints.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -eu
+
+runFromHook=$1
+[[ $runFromHook == "true" ]] && cd ..
+
+. ./bin/common.sh
+
+gatekeeperArtifactsPath="./values/gatekeeper/constraints"
```

Comment:
why are you generating these here? Can this be done in a /tmp/* folder? I don't propose generating code in the repo folders.